ffi/athcon/bindings/rust/athcon-client/tests/integration_tests.rs

```diff
@@ -8,7 +8,7 @@ use athcon_client::{
 };
 use athcon_vm::{MessageKind, Revision, StatusCode, StorageStatus};
 use athena_interface::payload::ExecutionPayload;
-use athena_interface::ADDRESS_ALICE;
+const ADDRESS_ALICE: Address = [0xAAu8; 24];
```

Comment:
Why redefine this here?

Reply:
I'm slowly removing all uses of the dummy addresses defined in the `athena-interface` with the end goal of removing them entirely. Every test should define the addresses it wants to work with itself.